sampler_remote.go

```diff
@@ -142,12 +143,17 @@ func (s *RemotelyControlledSampler) Equal(other Sampler) bool {
 }
 
 func (s *RemotelyControlledSampler) pollController() {
-	ticker := time.NewTicker(s.samplingRefreshInterval)
+	expBackOff := backoff.NewExponentialBackOff()
```

Comment:
Would you be able to document this somewhere? Like, what are the default backoff parameters?